exchanges/bybit/bybit_wrapper.go

```diff
@@ -74,6 +74,32 @@ func (by *Bybit) SetDefaults() {
 	}
 
 	by.Features = exchange.Features{
+		CurrencyTranslations: currency.NewTranslations(
+			map[currency.Code]currency.Code{
+				currency.NewCode("10000000AIDOGE"):  currency.AIDOGE,
+				currency.NewCode("1000000BABYDOGE"): currency.BABYDOGE,
+				currency.NewCode("1000000MOG"):      currency.NewCode("MOG"),
+				currency.NewCode("10000COQ"):        currency.NewCode("COQ"),
+				currency.NewCode("10000LADYS"):      currency.NewCode("LADYS"),
+				currency.NewCode("10000NFT"):        currency.NFT,
+				currency.NewCode("10000SATS"):       currency.NewCode("SATS"),
+				currency.NewCode("10000STARL"):      currency.STARL,
+				currency.NewCode("10000WEN"):        currency.NewCode("WEN"),
+				currency.NewCode("1000APU"):         currency.NewCode("APU"),
+				currency.NewCode("1000BEER"):        currency.NewCode("BEER"),
+				currency.NewCode("1000BONK"):        currency.BONK,
+				currency.NewCode("1000BTT"):         currency.BTT,
+				currency.NewCode("1000FLOKI"):       currency.FLOKI,
+				currency.NewCode("1000IQ50"):        currency.NewCode("IQ50"),
+				currency.NewCode("1000LUNC"):        currency.LUNC,
+				currency.NewCode("1000PEPE"):        currency.PEPE,
+				currency.NewCode("1000RATS"):        currency.NewCode("RATS"),
+				currency.NewCode("1000TURBO"):       currency.NewCode("TURBO"),
+				currency.NewCode("1000XEC"):         currency.XEC,
```

Comment:
Unlike XBT->BTC I don't consider 10000000AIDOGE to be the same as 1 AIDOGE. That seems more like this function is about returning the underlying currency of a futures contract currency, not translating it. If this is about returning underlying currencies I think things should be named more towards that as we already have existing language and features for that

Reply:
I disagree the preceding number is just a price multiplier. You are still trading the same asset. It's not really about returning underlying currency as its converting a specific representation into its base currency. The primary goal of the function is to enable a more seamless matching between spot and futures or across exchanges.

Either way though, the current design allows a client to externally modify this list and if you still deem it a pinch point I will RM these translations and I will have them externally supported.

Reply:
I don't know why I came to that conclusion the other day because I've dealt with them being price multipliers myself 🤙 Fine to keep 🌈